### ISSUE TITLE
Stale bot: closing feature requests and issues ?

### ISSUE THREAD

**Issue by uffejakobsen** [eranif/codelite#2474]
Today i found myself having to do phony comments on some of my feature requests and issues
This in order to prevent som "stale bot" from closing the feature requests and issues

It the "stale bot" something new ?
It seems to be a bad idea to have en automatic bot close feature request and issues

**Comment by eranif** [comment]
Yes. Agreed. I should probably tune it to avoid issues with specific labels :)

**Comment by uffejakobsen** [comment]
bad news - your adjustments did not work

stale bot is still active as of today

I just had to put another bunch of phony comments into my open issues to prevent the stale bot from closing them

**Comment by uffejakobsen** [comment]
Also if you really want to keep the stale bot - having a 5 day notice is not really enough - holiday season is approaching

**Comment by eranif** [comment]
Hey,
Stale bot now will not close issues with "Feature" label on them.

**Comment by uffejakobsen** [comment]
@eranif so what to do with issues that are not really feature requests but merely existing functionality that does not work ?

**Comment by eranif** [comment]
I have added another label `important` -> let's use that to mark such issues

**Comment by uffejakobsen** [comment]
@eranif erhm it seems that I cannot set labels...

**Comment by uffejakobsen** [comment]
once again....

This is a preventive action - this issue is still valid.
I'm making this comment to prevent the "stale bot" from closing it
It seems to be a bad idea to have en automatic bot close issues

This is a sort of a paradox - as this issue is not solved :-D

**Comment by eranif** [comment]
Adding the `important` label to issues will prevent the stale bot from closing these items

**Comment by AJenbo** [comment]
@eranif does stale bot only ignore `important` and `feature`?

**Comment by Eric-01** [comment]
Currently the Stale bot omits issues with the following tags:
* pinned
* security
* Feature
* Important

The tags and several other things can be adjusted in [~/.github/stale.yml](https://github.com/eranif/codelite/blob/master/.github/stale.yml).

**Comment by uffejakobsen** [comment]
@Eric-01 thanks for the explanation

I'll vote for raising "daysUntilClose" 
Currently it is 5 - during vacation periods that would result in subjects beeing closed - a value around woud be more reasonable 20
